src/pages/Editor.vue

```diff
@@ -40,14 +40,16 @@
                 class="z-0"
               ></video-player>
               <!-- maximize button -->
-              <icon-button
-                v-if="showItemModal"
-                :titleConfig="minimizeButtonTitleClass"
-                :buttonClass="minimizeButtonClass"
-                @click="toggleMinimize"
-                class="absolute z-20 top-2 right-2 btn"
-              >
-              </icon-button>
+              <transition name="maximize-btn-transition">
+                <icon-button
+                  v-if="showItemModal && isModalMinimized"
+                  :titleConfig="maximizeButtonTitleClass"
+                  :buttonClass="maximizeButtonClass"
+                  @click="maximize"
```

Comment:
a slightly more descriptive name of the function would be better: `maximizeModal`